layouts/partials/how-pulumi-works-lite.html

```diff
@@ -0,0 +1,54 @@
+<section id="how-it-works" class="bg-purple-500 py-16 px-4 md:px-0">
```

Comment:
Why remove the CTAs here? It'd also be preferable, since this is otherwise identical to the how-pulumi-works partial, to have that partial take an instruction to hide the elements we want to hide, rather than duplicate everything.